channels_api/bindings.py

```diff
@@ -78,8 +78,13 @@ def _group_name(self, action, id=None):
         else:
             return "{}-{}".format(self.model_label, action)
 
-    def has_permission(self, action, pk, data):
-        return True
+    def has_permission(self, user, action, pk):
+        permissions = True
+
+        for class_permission in self.permission_classes:
+            permission = class_permission().has_permission(user, action, pk)
+            permissions = permission and permissions
+        return permissions
 
```

Comment:
If you want to make the change, you'll have to make another commit and push it as well.